civihr_default_theme/scss/pages/_my-leave-request.scss

```diff
@@ -0,0 +1,19 @@
+.chr_leave-request-form__group {
+  margin-bottom: 15px !important;
+}
+
+.chr_leave-request-form__table {
+  @extend .chr_leave-request-form__group;
```

Comment:
Always avoid using `@extend`, it's a nasty sass directive whose output is unpredictable